test/test_helper.bash

```diff
@@ -46,16 +24,66 @@ local_teardown() {
   true
 }
 
+# This function sets up a local test fixture, i.e. a completely
+# fresh and isolated Bash-it directory. This is done to avoid
+# messing with your own Bash-it source directory.
+# If you need this, call it in your .bats file's `local_setup` function.
+setup_test_fixture() {
+  mkdir -p "$BASH_IT"
+  lib_directory="$(cd "$(dirname "$0")" && pwd)"
+  # Use rsync to copy Bash-it to the temp folder
+  # rsync is faster than cp, since we can exclude the large ".git" folder
+  rsync -qavrKL -d --delete-excluded --exclude=.git --exclude=enabled $lib_directory/../../../.. "$BASH_IT"
+
+  rm -rf "$BASH_IT"/enabled
+  rm -rf "$BASH_IT"/aliases/enabled
+  rm -rf "$BASH_IT"/completion/enabled
+  rm -rf "$BASH_IT"/plugins/enabled
+
+  mkdir -p "$BASH_IT"/enabled
+  mkdir -p "$BASH_IT"/aliases/enabled
+  mkdir -p "$BASH_IT"/completion/enabled
+  mkdir -p "$BASH_IT"/plugins/enabled
+
+  # Some tests use the BASH_IT_TEST_HOME variable, e.g. install/uninstall
+  export BASH_IT_TEST_HOME="$TEST_TEMP_DIR"
+}
+
 setup() {
+  # Temporarily using `mktemp` directly, since the bats-file function
```

Comment:
Consider adding a `FIXME` until `bats-core` project provides `bats-file` with macOS support.

Reply:
Good point! I'll try to switch to the other fork of bats-file, which seems to resolve the issue.

Reply:
I've switched to the tralston/bats-file fork now - this fixes the issue with `temp_make` on macOS.